projects/ui/src/components/Swap/Actions/Swap.tsx

```diff
@@ -465,7 +447,7 @@ const SwapForm: FC<FormikProps<SwapFormValues> & {
               /// user has no balance of the input.
               || noBalance
               /// No way to quote for this pathway
-              || !pathwayCheck
+              // || !pathwayCheck
```

Comment:
Nit: We can just remove it instead of commenting it out

Reply:
Great, will leave for now and remove on next edit.